__tests__/test.js

```diff
@@ -79,12 +78,47 @@ describe("GET /api/articles/:article_id", () => {
 		.get("/api/articles/99")
 		.expect(404)
 		.then((response) => {
-			console.log(response.body)
 			expect(response.body).toEqual({});
 		});
 	});
+});
 
-
+	describe("GET /api/articles", () => {
+		test("STATUS 200: responds with status code 200, get all articles", () => {
+			return request(app)
+			.get("/api/articles")
+			.expect(200)
+			});
+	test("status 200: Responds with an articles array of article objects, each of which should have the following properties", () => {
+		return request(app)
+		.get("/api/articles")
+		.expect(200)
+			.then(({body}) => {
+			const articles = body.articles.rows;
+			articles.forEach((article) => {
+			expect(article).toMatchObject({
+				article_id: expect.any(Number),
+				author: expect.any(String),
+				title: expect.any(String),
+				body: expect.any(String),
+				created_at: expect.any(String),
+				votes: expect.any(Number),
+				article_img_url: expect.any(String),
+				comment_count: expect.any(String)
```

Comment:
The actual number of comments could be checked here to make sure the query works well.